homeassistant/components/android_ip_webcam/sensor.py

```diff
@@ -157,11 +174,14 @@ def __init__(
         self.entity_description = description
         super().__init__(coordinator)
 
-    @callback
-    def _handle_coordinator_update(self) -> None:
-        """Update value and unit of sensor."""
-        (
-            self._attr_native_value,
-            self._attr_native_unit_of_measurement,
-        ) = self.entity_description.value_fn(self._ipcam)
-        self.async_write_ha_state()
+    @property
+    def native_value(self) -> StateType:
+        """Return native value of sensor."""
+        value, _ = self.entity_description.value_fn(self.entity_description.key)
+        return value
+
+    @property
+    def native_unit_of_measurement(self) -> str | None:
+        """Return native unit of measurement of sensor."""
+        _, unit = self.entity_description.unit_fn(self.entity_description.key)
```

Comment:
Make it a single return value.
```suggestion
        unit = self.entity_description.unit_fn(self.entity_description.key)
```